CSS/style.css

```diff
@@ -0,0 +1,159 @@
+:root {
+  --aliceblue: #e5e5e5;
+  --white: #fff;
+  --primary-color: #6070ff;
+  --dark-blue: #344563;
+  --skyblue: #7f8cff;
+  --dark-grey: #505f79;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+body {
+  background: var(--aliceblue);
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+}
+
+.container {
+  min-width: 80%;
+  margin: 0 auto;
+  padding: 20px;
+}
+
+header {
+  position: absolute;
+  width: 375px;
+  height: 80vh;
+  background: var(--white);
+  background-image: url(../images/Img/header-shapes\ mobile@2x.png);
+  background-repeat: no-repeat;
+}
+
+/* images/Img/header-shapes\ mobile@2x.png */
+
+/* header nav */
+.top-nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+  margin: 20px 0 10px 5px;
+  padding: 20px;
+}
+
+.top-nav ul li {
+  margin: 2px 0 0 -45px;
+  list-style-type: none;
+}
+
+/* mylogo */
+.logo {
+  width: 76px;
+  height: 20px;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: var(--primary-color);
+  padding: 4px;
+  margin-left: 15px;
+}
+
+.vector {
+  position: absolute;
+  border: 2px solid #acb7c3;
+}
+
+/*  Section header section  */
+
+.greeting {
+  width: 327px;
+  height: 146px;
+  font-weight: 400;
+  font-size: 35px;
+  line-height: 36px;
```

Comment:
- [OPTIONAL] Good job fixing the CSS file path, however, I suggest modifying your style, removing the margin-left and top from this page and also making your page responsive to make your project look more like the Figma design and professional :recycle: 
![Screenshot from 2023-03-29 15-46-23](https://user-images.githubusercontent.com/97834160/228561443-1e418773-359c-481b-8bdc-c22deccaaf99.png)